### ISSUE TITLE
GHC 8?

### ISSUE THREAD

**Issue by FranklinChen** [mgajda/json-autotype#12]
I noticed `json-autotype` is no longer in the current Stackage nightly for GHC 8. https://www.stackage.org/nightly-2016-06-02

**Comment by FranklinChen** [comment]
Stackage LTS 8 is now finally out, and `json-autotype` currently does not seem to build:

```console
$ stack install json-autotype --resolver lts-8

Error: While constructing the build plan, the following exceptions were encountered:

In the dependencies for json-autotype-1.0.15:
    aeson-1.0.2.1 must match >=0.7 && <0.12 (latest applicable is 0.11.3.0)
    lens-4.15.1 must match >=4.1 && <4.15 (latest applicable is 4.14)

Plan construction failed.
```

**Comment by mgajda** [comment]
Should be fixed as of release 1.0.16.
